React-2/Week-1/my-app/app/pages/blog_website/[id].js

```diff
@@ -0,0 +1,19 @@
+// pages/blogs/[id].js
+import { useRouter } from "next/router";
+
+export default function BlogPost() {
+  const router = useRouter();
+  const { id } = router.query;
```

Comment:
1. To get query from url we use `useSearchParams`
2. To get params from path (as it suppose to be here) we use either `await params` for Server Components or `useParams` for Client Components
3. Router doesn't have `.query`, unfortunatelly

Reply:
Would you kindly review my code once more?